### PR TITLE
AP-143: Remove all mention of the refresh token from the tech docs and the diagrams

### PR DESCRIPTION
**Pull request by pauldougan**
[AP-143](https://govukverify.atlassian.net/browse/AP-143)

## Why

We are deprecating the use of the refresh token 

## What

Remove all mention of the refresh token from the tech docs and the diagrams

## Technical writer support

Yes, this is a significant change, although almost exclusively deletion of content.

## How to review

- Look at the tech docs as a whole and ensure they still make sense and are internally consistent
- ensure that there are no search results for refresh

[AP-143]: https://govukverify.atlassian.net/browse/AP-143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by philf999 — APPROVED** [review]
Looks good to me

**Review by charleemg — APPROVED** [review]
This content does not mention refresh tokens anymore.

**Comment by pauldougan** [comment]
@PippaClarkGDS I have removed the rogue arrow head as requested

**Comment by pauldougan** [comment]
@huw we are in need of a code owner review on this one pls

**Comment by huw** [comment]
@huwd 😘

**Comment by pauldougan** [comment]
Hi @sabrinaharris we have a tech docs pr in need of your final blessing

File: README.md
Comment (sabrinaharris) [inline README.md:106]:
````suggestion
## Making changes to a diagram

To make changes to a diagram, you'll need to edit the `drawio` files in `source/images/originals` using the draw.io desktop app.

### Install draw.io desktop app

1. Install the draw.io desktop app using Homebrew with the following command: 

```
brew cask install drawio
```

2. Alias the binary so that you can just type `draw.io` at the command line. Use the following command to do this:

```
alias=draw.io='/Applications/draw.io.app/Contents/MacOS/draw.io'
```

### Edit and publish the diagram

Follow these steps:
1. Create and modify the diagram using [draw.io](https://about.draw.io/).
1. Store the draw.io original file in `source/images/originals`.
1. Publish the diagram as a [scalable vector graphic (SVG)](https://www.w3.org/Graphics/SVG/).
1. Store the SVG in the `source/images` folder.

### Useful commands

Update a diagram:

```
draw.io source/images/originals/top-level-technical-diagram.drawio
```

Generate SVG versions of the diagrams:

```
draw.io -x -o source/images/top-level-technical-diagram.svg source/images/originals/top-level-technical-diagram.drawio

draw.io -x -o source/images/technical-flow-diagram.svg source/images/originals/technical-flow-diagram.drawio
```
````

Comment (sabrinaharris) [reply README.md:106]: Have suggested we could leave the README update out of this PR to get the refresh token changes in ASAP.

Comment (pauldougan) [reply README.md]: Hi @sabrinaharris I have pulled the README update and will make another PR to tackle this after the content has shipped. Its only me making the diagram changes at the moment so not a problem. Longer term it needs to be documented

**Review by sabrinaharris — APPROVED** [review]

File: README.md
Comment (sabrinaharris) [reply README.md]: Thanks very much @pauldougan!

Comment (sabrinaharris) [reply README.md]: Created [issue for README changes ](https://govukverify.atlassian.net/jira/software/c/projects/AP/boards/452?selectedIssue=AP-166) 👍

**Pull request merged**
